CMakeLists.txt

```diff
@@ -93,11 +84,18 @@ else()
 endif()
 
 add_library(onnx_proto ${PROTO_SRCS} ${PROTO_HDRS})
-add_dependencies(onnx_proto protobuf::libprotobuf)
-target_compile_options(onnx_proto PRIVATE /WX- ${DLLEXPORT_OPTION})
+target_include_directories(onnx_proto PUBLIC "${CMAKE_CURRENT_BINARY_DIR}" "${PROTOBUF_INCLUDE_DIRS}")
```

Comment:
${CMAKE_CURRENT_BINARY_DIR} is not needed, I think.

Reply:
@linkerzhang I think it's needed :-) Say if another lib depends on onnx_proto, then it will need to be able to find "pb.h" file.